Castle.Core-VS2017.sln

```diff
@@ -0,0 +1,71 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Wow, I just assumed Microsoft would have improved the solution file with the changes to project files.